classes/helper/HelperOptions.php

```diff
@@ -287,7 +290,9 @@ public function displayOptionTypeImage($key, $field, $value)
     public function displayOptionTypePrice($key, $field, $value)
     {
         echo $this->context->currency->getSign('left');
-        $this->displayOptionTypeText($key, $field, $value);
+        if (method_exists($this, 'displayOptionTypeText')) {
```

Comment:
Funny as method_exists will return true if method is private and PHP is lower than 7.4 :sweat_smile: